Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>node_modules/\r\ndist/\r\n.env\r\n.DS_Store\r\n#*.log\r\nbuild/\r\ncoverage/
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision a3cf1331a6b1443d51b65218b9b7829e1c7e4917)
+++ b/.gitignore	(date 1760840066620)
@@ -4,4 +4,4 @@
 .DS_Store
 #*.log
 build/
-coverage/
\ No newline at end of file
+coverage/
